src/config/experiment.js

```diff
@@ -1,22 +1,64 @@
-import { defaultBlockSettings } from './main';
-import { deepCopy } from '../lib/utils'
+import { deepCopy } from "../lib/utils"
+import localConfig from "./config.json"
+import { envConfig } from "./main"
+import path from "path"
+import { firestoreConfig } from "../firebase"
 
-// FIRST EXPERIMENT BLOCK SETTINGS
+const getConfig = async (participantID, studyID) => {
+  let experimentConfig = localConfig
+  if (envConfig.USE_ELECTRON) {
+    const app = window.require("electron").remote.app
+    const renderer = window.require("electron").ipcRenderer
+    const fs = window.require("fs")
```

Comment:
have you tested this with a built app? I just remembered the dev mode has lower security setting than a prod build